plugins/builds/metrics.js

```diff
@@ -2,6 +2,8 @@
 
 const boom = require('boom');
 const joi = require('joi');
+const { setDefaultTimeRange, validTimeRange } = require('../helper.js');
+const MAX_DAYS = 180; // 6 months
```

Comment:
can this is made a k8s env variable so that we could relax on fly ?

Reply:
what would it be under?

Reply:
API deployment config ? Not a big deal, but if we choose later to relax this env will be better